_docs_v7/Solver-Setup.md

```diff
@@ -42,6 +45,24 @@ SU2 is capable of dealing with different kinds of physical problems. The kind of
 
 Every solver has its specific options and we refer to the tutorial cases for more information. However, the basic controls detailed in the remainder of this page are the same for all problems.
 
+## Fluid Model ##
+
+For fluid simulations, a model defining the equation of state and thermodynamic properties of the fluid or mixture is required. This is selected using the `FLUID_MODEL` option in the config. Available fluid models in SU2 include:
+
+| Option Value | Description | 
+|---|---|
+|`STANDARD_AIR` | **Air model with ideal gas EOS** |
+|`IDEAL_GAS` | **Arbitrary fluid with ideal gas EOS** |
+|`VW_GAS` | **Arbitrary fluid with Vander-Waals EOS** |
+|`PR_GAS` | **Arbitrary fluid with Peng-Robinson EOS** |
```

Comment:
Does this little arrow symbol have a special meaning?  If so please explain in the text.

Reply:
Maybe I'm missing it, which arrow symbol? I don't see one.